src/binding/python/Series.cpp

```diff
@@ -30,7 +34,24 @@ using namespace openPMD;
 void init_Series(py::module &m) {
     py::class_<Series, Attributable>(m, "Series")
 
-        .def(py::init<std::string const&, AccessType>())
+        .def(py::init<std::string const&, AccessType>(),
+            py::arg("filepath"), py::arg("access_type"))
+#if openPMD_HAVE_MPI
+        .def(py::init([](std::string const& filepath, AccessType at, py::object &comm){
+            //! @todo perform mpi4py import test?
+            //!       required C-API init?
+            //!       https://bitbucket.org/mpi4py/mpi4py/src/3.0.0/demo/wrap-c/helloworld.c
+            // if( import_mpi4py() < 0 ) { here be dragons }
```

Comment:
trying to avoid `mpi4py/mpi4py.h` include:
Function generated during build. This just seems to import and perform tests on functions and type signatures for binary compatebility. We can probably skip it and do it with `pybind11` methods.